src/LineBot/RichMenu/RichMenuSize.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright 2017-2018 Dirk Lemstra (https://github.com/dlemstra/line-bot-sdk-dotnet)
+//
+// Dirk Lemstra licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Line
+{
+    /// <summary>
+    /// RichMenuSize object which contains the width and height of the rich menu displayed in the chat.
+    /// Rich menu images must be one of the following sizes: 2500x1686px or 2500x843px.
+    /// </summary>
+    public class RichMenuSize
```

Comment:
This needs an `IRichMenuSize` interface and that should be used in the `IRichMenu` interface instead of the class.